tests/test_run_reference_vmm.py

```diff
@@ -222,7 +226,24 @@ def run_cmd_inside_vm(cmd, vmm_process, prompt, timeout=5):
 
        Note: `cmd` and `prompt` should be a `bytes` object and not `str`.
     """
-    cmd = cmd.strip() + b'\r\n'
+    cmd = cmd.strip()
+    
+    # The max capacity of the buffer for serial console is 64.(https://github.com/rust-vmm/vm-superio/blob/282bae92878936086606715412494fa81151efba/crates/vm-superio/src/serial.rs#L34)
+    # So we split the command at 63 bytes. 1 extra is reserved for \r at the end.
+    # empty string is of size 33. sys.getsizeof(b'') = 33.
```

Comment:
Hmmm, this is rather strange. Why would the empty string require 33 bytes? Do you know why that is the case?